pkg/services/accesscontrol/roles.go

```diff
@@ -83,10 +95,12 @@ var (
 		},
 	}
 
-	usersOrgEditRole = RoleDTO{
-		Name:    usersOrgEdit,
-		Version: 1,
-		Permissions: ConcatPermissions(usersOrgReadRole.Permissions, []Permission{
+	orgUsersWriterRole = RoleDTO{
+		Name:        orgUsersWriter,
+		DisplayName: "Organization users writer",
+		Description: "Read, add, remove and update role for users in organization",
```

Comment:
```suggestion
		DisplayName: "Organization users writer",
		Description: "Read, add, remove, or update a role for users in organization",
```
"Organization users writer" is odd. We can give this a bit more thought as dev progresses.

Can the user have more than one role? (I think so, but if they can have more than one role, change from "a role" to "a role or roles".

"in organization" == "in [an, the, your] organization"?

Reply:
Okey so here there will be some confusion. This role would allow a user to set builtin role for other people in the org (Admin, Editor, Viewer). It is true that a user can have several fgac roles but a user can only have one builtin.